geoips/plugins/modules/output_checkers/geotiff.py

```diff
@@ -131,16 +133,17 @@ def outputs_match(plugin, output_product, compare_product):
 
     # subimg_retval = subprocess.call(call_list)
     if retval != 0:
-        LOG.interactive("    *****************************************")
-        LOG.interactive("    *** BAD geotiffs do NOT match exactly ***")
-        LOG.interactive("    ***   output_product: %s ***", output_product)
-        LOG.interactive("    ***   compare_product: %s ***", compare_product)
-        LOG.interactive("    *****************************************")
```

Comment:
Is there a reason you removed the *** at the beginning of the messages?  I kind of like the ***, do you mind if I put it back in?

Reply:
That seemed to break black.  I guess the "*** " could also just be added at the beginning of each message line in the log_with_emphasis function?

Reply:
I think we should just add "*** " to the log_with_emphasis functin. Will add comment there.